presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -100,18 +99,11 @@
     TableHandle getAlternativeTableHandle(Session session, TableHandle tableHandle, PartitioningHandle partitioningHandle);
 
     /**
-     * Experimental: if true, the engine will invoke pushdownFilter instead of getLayout.
-     *
-     * This interface can be replaced with a connector optimizer rule once the engine supports these (#12546).
-     */
-    boolean isPushdownFilterSupported(Session session, TableHandle tableHandle);
-
-    /**
-     * Experimental: returns table layout that encapsulates the given filter.
-     *
-     * This interface can be replaced with a connector optimizer rule once the engine supports these (#12546).
+     * Experimental: if true, the engine will invoke getLayout otherwise, getLayout will not be called.
+     * This function remains to ensure backwards compatibility, it needs to be removed.
```

Comment:
Explain more how to use `ConnectorPlanOptimizer` as the way to pushdown compute.

Reply:
Let me know if the description I added is sufficient.